### DISCUSSION TITLE
When using components packaged with Lit, there is no property auto-completion when referenced as an npm package in Vue project

### DISCUSSION THREAD

**Discussion by Sweet-KK** [lit/lit#4480] [Q&A]
When using components packaged with Lit, even though I've added TypeScript types, there is no property auto-completion when referenced as an npm package in a Vue project. Hovering the mouse over it does show the type definitions. How can I resolve this issue with editor suggestions?

**Comment by Sweet-KK** [comment]
<img width="542" alt="image" src="https://github.com/lit/lit/assets/29600932/136e3877-f162-4915-b297-a68f00cd50ac">

**Comment by augustjk** [comment]
I'm not too familiar with the Vue ecosystem but are you using an extension like https://marketplace.visualstudio.com/items?itemName=Vue.volar ?

And mapped the types to vue's GlobalComponents interface? https://vuejs.org/guide/extras/web-components#web-components-and-typescript
